metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -202,6 +203,7 @@
 client_jaas_path = metron_home + '/client_jaas.conf'
 client_jaas_arg = '-Djava.security.auth.login.config=' + metron_home + '/client_jaas.conf'
 enrichment_topology_worker_childopts = client_jaas_arg if security_enabled else ''
+profiler_topology_worker_childopts = client_jaas_arg if security_enabled else ''
```

Comment:
As it stand, setting this property in Ambari won't have any effect.  There needs to be an additional step that reads from the config and appends.  See enrichment_topology_worker_childopts as an example.

Reply:
whoops, missed a step.  Thanks!